scripts/load_testing/foney.py

```diff
@@ -0,0 +1,63 @@
+# thanks to https://gist.github.com/ifnull/a32fcc90cd60f6e85b57
```

Comment:
We couldn't get faker to do it clean?

Reply:
This generates real ones everytime. Faker is only capable of filling a pattern with random digits. It doesn't have any logic as to what a valid US phone number looks like. This may be useful code to push upstream to the Py/Ruby faker libs.

Reply:
Cool.